chapter02/jmw/src/main/java/spring/training/services/BankService.java

```diff
@@ -0,0 +1,43 @@
+
+package spring.training.services;
+
+import java.math.BigDecimal;
+
+import spring.training.model.Account;
+import spring.training.model.AccountTransaction;
+
+public class BankService {
+
+	static BigDecimal minBalance = new BigDecimal( 0 );
+
+	public boolean debit( Account toAccount, BigDecimal amount ) {
+		boolean success = false;
```

Comment:
I feel, returning a status of operation not a good design. The users can simply ignore them, time to time and later it is difficult to track down the actual issue.